app/__init__.py

```diff
@@ -0,0 +1,7 @@
+from sdx.common.logger_config import logger_initial_config
+
+from app import settings
+
+__version__ = "1.0.0"
```

Comment:
Yeah probably should, i'll change, though i think we need a discussion on our semantic versioning. Something for the next tech meeting?